src/main/java/pl/psnc/dei/service/TranscriptionPlatformService.java

```diff
@@ -208,7 +209,7 @@ public JsonObject fetchTranscriptionUpdate(Transcription transcription) {
 					}
 				})
 				.block();
-
+		// TODO retrieve just the transcription that we need
 		return JSON.parse(response);
```

Comment:
Here we need to do it in this task but I was waiting for an answer from Marius about possibilities other than parsing the whole enrichments response. Since there is no info from him implement parsing and retrieving just the transcription for which the method was called.